client/app/bundles/course/admin/pages/RagWiseSettings/components/FolderTab.tsx

```diff
@@ -42,12 +44,20 @@ const FolderTab: FC<FolderTabProps> = (props) => {
     [],
     sortItems,
   );
-  const isFolderExpanded = useAppSelector(getExpandedSettings);
+  const isFolderExpanded = useAppSelector(getFolderExpandedSettings);
 
   return (
     <CollapsibleList
       collapsedByDefault
       forceExpand={isFolderExpanded}
+      headerAction={
+        <div className="pr-7">
```

Comment:
why use margin for elsewhere (mr-1) and padding here (pr-7)?
Also, instead of introducing an additional div element, allow `MaterialKnowledgeBaseSwitch` to accept `className` to extend any additionally provided classNames.

Reply:
![image](https://github.com/user-attachments/assets/87dd74f0-490d-4ed0-9aa3-27ed129fa360)
using mr-1 here will lead to buttons being not aligned. I can change all pr-7 to mr-7 for consistency

Reply:
added classname to `MaterialKnowledgeBaseSwitch `and `ForumKnowledgeBaseSwitch`